### DISCUSSION TITLE
How can I update Existing Wallet Logo and Name

### DISCUSSION THREAD

**Discussion by chainpioneer1** [WalletConnect/walletconnect-monorepo#1211] [Developer Support] [answered]
How can I update our wallet logo and name on walletconnect registry?
I have already created account on https://cloud.walletconnect.com/app/registry
But on site, there is no option to update existing wallet logo and metadata.
How can I do it?
Thank you.

**Comment by looter** [comment, marked as answer]
Thanks for your question. If your application is published you'll be able to update the logo and the metadata and republish it. If it isn't you'll need to wait until it's approved or rejected to make those changes.
